include/swift/AST/ExistentialLayout.h

```diff
@@ -28,6 +28,8 @@ namespace swift {
   class ProtocolCompositionType;
 
 struct ExistentialLayout {
+  enum Kind { ClassExistential, ErrorExistential, OpaqueExistential };
```

Comment:
You could reasonably drop `Existential` from these names.

Reply:
Sure, I can do that.